java/java.lsp.server/vscode/src/extension.ts

```diff
@@ -1311,6 +1322,20 @@ export function deactivate(): Thenable<void> {
     return stopClient(client);
 }
 
+function collectDocumentSelectors(): TextDocumentFilter[] {
+    const selectors = [];
+    for (const extension of vscode.extensions.all) {
+        const contributesSection = extension.packageJSON['contributes'];
+        if (contributesSection) {
+            const documentSelectors = contributesSection['nbls.documentSelectors'];
```

Comment:
reference to the previous `nbls.` prefix (schema was changed to `netbeans.`

Reply:
OOPS - fixed.